sqlserver/datadog_checks/sqlserver/statements.py

```diff
@@ -120,9 +140,11 @@
         ELSE statement_end_offset
     END - statement_start_offset) / 2) + 1) AS statement_text,
     SUBSTRING(qt.text, 1, {proc_char_limit}) as text,
+    OBJECT_NAME(sproc_object_id, sproc_database_id) as object_name,
+    OBJECT_SCHEMA_NAME(sproc_object_id, sproc_database_id) as schema_name,
```

Comment:
Can we give it a more specific name, maybe something like `sproc_name` and `sproc_schema_name`?